taxonomies/food/ingredients.txt

```diff
@@ -71088,13 +71104,13 @@ wikipedia:en:https://en.wikipedia.org/wiki/Green_tea
 # ingredient/green-tea has 902 products in 14 languages @2019-04-21
 
 <en:green tea
-en:matcha, matcha green tea
+en:matcha, matcha green tea, matcha tea
 bg:матча
 ca:té matcha, té matcha verd
 da:matcha
 de:Matcha
 es:té matcha, té matcha verde
-fr:matcha, thé vert matcha
+fr:matcha, thé vert matcha, matcha
```

Comment:
```suggestion
fr:matcha, thé vert matcha
```
it's the primary French name, too :)

Reply:
I modified it, I wanted to write thé matcha